hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/BlockRecordStreamConfig.java

```diff
@@ -38,15 +40,15 @@
  */
 @ConfigData("hedera.recordStream")
 public record BlockRecordStreamConfig(
-        @ConfigProperty(defaultValue = "true") boolean enabled,
-        @ConfigProperty(defaultValue = "hedera-node/data/recordStreams") String logDir, // NODE LOCAL PROPERTY
-        @ConfigProperty(defaultValue = "sidecar") String sidecarDir, // NODE LOCAL PROPERTY
-        @ConfigProperty(defaultValue = "2") @Min(1) int logPeriod, // NETWORK PROPERTY
-        @ConfigProperty(defaultValue = "5000") @Min(1) int queueCapacity, // COULD BE NODE LOCAL PROPERTY
-        @ConfigProperty(defaultValue = "256") @Min(1) @Max(1024) int sidecarMaxSizeMb, // NETWORK PROPERTY
-        @ConfigProperty(defaultValue = "6") @Min(1) int recordFileVersion, // NETWORK PROPERTY
-        @ConfigProperty(defaultValue = "6") @Min(1) int signatureFileVersion, // NETWORK PROPERTY
-        @ConfigProperty(defaultValue = "true") boolean compressFilesOnCreation, // NOT SURE
-        @ConfigProperty(defaultValue = "256") @Min(1) @Max(4096) int numOfBlockHashesInState, // NETWORK PROPERTY
-        @ConfigProperty(defaultValue = "concurrent")
+        @ConfigProperty(defaultValue = "true") @NodeProperty boolean enabled,
```

Comment:
If I understand what the record stream controls, I think we would want the `enabled` and `logPeriod` values to be network controlled so all nodes produce record streams and the record streams are reasonably compatible.

Reply:
No change.